playwright-tests/tests/stake-delegation/stake-delegation.spec.js

```diff
@@ -999,7 +1003,7 @@ test.describe("Lockup staking", function () {
               actions: [
                 {
                   method_name: "deposit_and_stake",
-                  args: "eyJhbW91bnQiOiIxMDcwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMCJ9",
+                  args: "eyJhbW91bnQiOiI3MjAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwIn0=",
```

Comment:
It would be better to show the actual args here and use a function for converting to base64. Makes the code more readable

Reply:
yeah good suggestion, I wasn't able to reproduce the same base64 w/o Big, but installed it.